jmix-security/security/src/main/java/io/jmix/security/model/RoleSourceEnum.java

```diff
@@ -14,21 +14,21 @@
  * limitations under the License.
  */
 
-package io.jmix.securityflowui.model;
+package io.jmix.security.model;
 
 import io.jmix.core.metamodel.datatype.EnumClass;
 
 import org.springframework.lang.Nullable;
 
 
-public enum RoleSource implements EnumClass<String> {
+public enum RoleSourceEnum implements EnumClass<String> {
```

Comment:
Probably it's better to name it `RoleSourceType` because we usually don't use `Enum` in enum names.

Reply:
Done